installation/aws-container.md

```diff
@@ -10,6 +10,11 @@ Currently, the [AWS for Fluent Bit](https://github.com/aws/aws-for-fluent-bit) i
 * [Amazon Kinesis Firehose](https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit)
 * [Amazon Kinesis Streams](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit)
 
+Fluent Bit includes CloudWatch Logs plugin named `cloudwatch_logs` and Kinesis Firehose plugin named `kinesis_firehose` which are higher performance than Go plugins.
+
+* [Amazon CloudWatch](https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch)
+* [Amazon Kinesis Data Firehose](https://docs.fluentbit.io/manual/pipeline/outputs/firehose)
```

Comment:
@SpringMT we also now have a core kinesis_streams plugin too: https://docs.fluentbit.io/manual/pipeline/outputs/kinesis

Please add that one and then we can merge this.

Reply:
Added `kinesis_streams` explanation.
Thx!